WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -79,8 +80,9 @@ public static int Main(string[] args)
 						.AfterSetup(_ => ThemeHelper.ApplyTheme(Global.UiConfig.DarkModeEnabled ? Theme.Dark : Theme.Light))
 						.StartWithClassicDesktopLifetime(args);
 				}
-				catch (OperationCanceledException ex)
+				catch (WasabiAlreadyRunningException ex)
```

Comment:
Why do we need to change this?

Reply:
This was as intended. If the other instance is running and we are able to signal it, that means we just quit silently and the other instance will pop up. With your change, the other instance will pop up, plus we will get a crash report.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/4842#discussion_r692851104